src/documents/CaseStudyExemplar.md

```diff
@@ -72,6 +83,7 @@ In order to use scripts in EPUB, CDATA blocks need to be added. However this wil
 </script>
 ```
 
+<!-- markdownlint-disable ol-prefix -->
```

Comment:
According to https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md029---ordered-list-item-prefix the rules should still pass if the code blocks are indented. Could you see if you can make it work without disabling the rule itself?